test/NuGet.Core.Tests/NuGet.Configuration.Test/PackageSourceProviderTests.cs

```diff
@@ -93,28 +93,28 @@ public void LoadPackageSources_LoadsCredentials()
             var configContent = @"<?xml version='1.0' encoding='utf-8'?>
 <configuration>
   <apikeys>
-    <add key='https://www.nuget.org' value='removed' />
-    <add key='https://www.myget.org/F/somecompanyfeed-unstable/' value='removed' />
-    <add key='https://www.myget.org/F/somecompanyfeed/' value='removed' />
-    <add key='https://www.myget.org/F/somecompanyfeed-unstable/api/v2/package' value='removed' />
-    <add key='https://www.myget.org/F/somecompanyfeed/api/v2/package' value='removed' />
-    <add key='https://www.myget.org/F/somecompanyfeed-unstable/api/v2/' value='removed' />
-    <add key='https://nuget.smbsrc.net/' value='removed' />
+    <add key='https://a.test' value='removed' />
```

Comment:
To anyone else reviewing who, like me, was concerned that `a.test` could be a real domain, luckily the IETF reserved the `.test` TLD for software testing, so it's not possible for these domains to point to anything real. https://en.wikipedia.org/wiki/.test